### DISCUSSION TITLE
View favorite  photos on Picture frame

### DISCUSSION THREAD

**Discussion by huff** [photoprism/photoprism#4245] [General]
I have a raspberry pi connected to a photo frame and it's currently running FullPageOS....

I have it configured to load my favorite photos via  "hostname:2342/library/favorites#&gid=1&pid=1" However I have to log in and click play and full screen for it to play all my favorite photos...

Is there a better way to do this without manual intervention?  Can I add a username and pw in the URL? and have it auto-play? Also, can the time between pictures be longer?

Thanks for any suggestions....

-D

**Comment by j-addante** [comment]
I am currently working on something to solve this! I discovered the [ImmichFrame](https://immichframe.dev/) project and wanted to make something similar for PhotoPrims. Its currently a WIP but it will be able to be the middle man between PhotoPrism and ImmichFrame apps! My project is linked below

https://gitlab.com/j-addante/photoprismframe
